src/components/ha-card.ts

```diff
@@ -27,23 +27,43 @@ class HaCard extends LitElement {
         color: var(--primary-text-color);
         display: block;
         transition: all 0.3s ease-out;
+        position: relative;
       }
-      .header:not(:empty),
-      .header::slotted(*) {
+
+      :host .card-header,
+      :host ::slotted(.card-header) {
         color: var(--ha-card-header-color, --primary-text-color);
         font-family: var(--ha-card-header-font-family, inherit);
         font-size: var(--ha-card-header-font-size, 24px);
         letter-spacing: -0.012em;
         line-height: 32px;
-        padding: 24px 16px 16px;
+        padding: 24px 16px 8px 16px;
```

Comment:
The 8px is only needed if there is a `paper-content` with their own padding. If the user is not using `paper-content`, making it 8px means any content has to compensate.

Reply:
Right.